parser-typechecker/src/Unison/Codebase/SqliteCodebase/Branch/Dependencies.hs

```diff
@@ -52,9 +51,9 @@ to' Dependencies {..} = Dependencies' (toList patches) (toList terms) (toList de
 
 fromBranch :: Applicative m => Branch m -> (Branches m, Dependencies)
 fromBranch (Branch c) = case c of
-  Causal.One _hh e -> fromBranch0 e
-  Causal.Cons _hh e (h, m) -> fromBranch0 e <> fromTails (Map.singleton h m)
-  Causal.Merge _hh e tails -> fromBranch0 e <> fromTails tails
+  Causal.One _hh _eh e -> fromBranch0 e
+  Causal.Cons _hh _eh e (h, m) -> fromBranch0 e <> fromTails (Map.singleton h m)
+  Causal.Merge _hh _eh e tails -> fromBranch0 e <> fromTails tails
   where
     fromTails m = ([(h, Branch <$> mc) | (h, mc) <- Map.toList m], mempty)
```

Comment:
Do we want to do anything more with _eh here?

Reply:
I took a look at where this function is used in the sync algorithm, and no I don't think so; the sync algorithm syncs the branch hash ID as part of syncing the causal hash, so I don't think adding it here provides any value.